main.tf

```diff
@@ -0,0 +1,148 @@
+module "label" {
+  source     = "github.com/cloudposse/terraform-terraform-label.git?ref=master"
+  attributes = "${var.attributes}"
+  delimiter  = "${var.delimiter}"
+  name       = "${var.name}"
+  namespace  = "${var.namespace}"
+  stage      = "${var.stage}"
+  tags       = "${var.tags}"
+}
+
+resource "aws_ecs_task_definition" "default" {
+  family                   = "${module.label.id}"
+  container_definitions    = "${var.container_definition_json}"
+  requires_compatibilities = ["${var.launch_type}"]
+  network_mode             = "${var.network_mode}"
+  cpu                      = "${var.task_cpu}"
+  memory                   = "${var.task_memory}"
+  execution_role_arn       = "${aws_iam_role.ecs_execution_role.arn}"
+  task_role_arn            = "${aws_iam_role.ecs_execution_role.arn}"
```

Comment:
we must pass:
```
  desired_count                      = "${var.desired_count}"
  deployment_maximum_percent         = "${var.deployment_maximum_percent}"
  deployment_minimum_healthy_percent = "${var.deployment_minimum_healthy_percent}"
```

Reply:
I added these to the `aws_ecs_service` - was missing the deployment_* ones... https://github.com/cloudposse/terraform-aws-ecs-alb-service-task/pull/1/files#diff-7a370d8342e7203b805911c92454f0f4R135